Content.Shared/Magic/SharedMagicSystem.cs

```diff
@@ -0,0 +1,66 @@
+using Content.Shared.Magic.Events;
+using Robust.Shared.Serialization.Manager;
+using JetBrains.Annotations;
+
+namespace Content.Shared.Magic;
+
+/// <summary>
+/// Handles using spells
+/// </summary>
+[UsedImplicitly]
+public abstract class SharedMagicSystem : EntitySystem
```

Comment:
need a client `MagicSystem` to run this properly since this is abstract

Reply:
Oops, seems like `net.fakelagmin` now defaults to 0 which made me assume that this worked. Seems like the events don't get raised on the client but I will reopen when I figure out how to fix this.